packages/syft/src/syft/service/api/api_service.py

```diff
@@ -394,7 +394,23 @@ def call(
         )
         if isinstance(custom_endpoint, SyftError):
             return custom_endpoint
-        return Ok(custom_endpoint.exec(context, *args, **kwargs))
+
+        exec_result = custom_endpoint.exec(context, *args, **kwargs)
+
+        if isinstance(exec_result, SyftError):
```

Comment:
Lets add types for these in our Tech Debt cycle, the user should be able to return UserSyftErrors that we can pass back as "good" results.

Also making it so they don't need to `import syft as sy` or other things in their code would be good.